libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -3402,9 +3402,10 @@ string YulUtilFunctions::conversionFunction(Type const& _from, Type const& _to)
 			{
 				// clear for conversion to longer bytes
```

Comment:
This comment also no longer applies - it clears for conversion to longer and shorter bytes types.